src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -879,11 +879,99 @@ tags:
             }
       ```
 
+      ### Iterating over all key value store keys
+
+      You can use `/state/key-value-store/keys` endpoint to iterate over all keys in certain key value store.
+      If the queried key value store contains more than one page of keys, a next_cursor will be returned, which can be used as cursor in the request to fetch further pages.
+
+      i.e:
+      ```
+      /state/key-value-store/keys
+
+      {
+        "key_value_store_address": "internal_keyvaluestore_tdx_2_1kzjd929eqlzd9n02uuj8jd48705vcrpvhv4mgxnaltrgystnca3qxk"
+      }
+      ```
+
+      Will respond with list of all keys (in both `hex` and `programmatic_json` form) and state version when they were last updated.
```

Comment:
Minor English article corrections:

```
      Will respond with a list of all the keys (in both `hex` and `programmatic_json` form) and the state version when they were last updated.
```

However(!) I'd suggest we say it's actually not all keys - it's importantly the *first page of keys*, and we should explain how it's ordered too.

Reply:
how about

```
Will respond with a first page of all keys (in both `hex` and `programmatic_json` form) and the state version when they were last updated. Returned keys are ordered by their last change, first key on a list will be one that was most recently updated/created. 
```

Reply:
I suggest this tweak, if you're happy with it? 
```
Will respond with a first page of the keys of the current entries of the key value store. This collection is ordered by the state version of the entry's last update, descending. In other words, the first entry on the list will be one that was most recently updated/created. Each item includes the entry's key (in both `hex` and `programmatic_json` form) and the state version when it was last updated.
```